src/bullsAndCows.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+exports.calculateBullsCows = (winNumber, playerNumber) => {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < 4; i++) {
```

Comment:
If you try to make a game with the possibility to change random number length, let's also change cycle length